common/lib/xmodule/xmodule/capa_base.py

```diff
@@ -643,17 +637,58 @@ def get_problem_html(self, encapsulate=True):
         demand_hints = self.lcp.tree.xpath("//problem/demandhint/hint")
         demand_hint_possible = len(demand_hints) > 0
 
+        # Get the current problem status and generate the answer notification and message.
+        answer_notification_message = None
+        answer_notification_type = None
+        progress = self.get_progress()
+
+        if progress is not None:
```

Comment:
As I mentioned, talk to @chris-mike about what should show when progress is None (which can happen for ungraded problems). I do think some sort of notification should show.

Also make sure that the ungraded case has test coverage.